.storybook/themes/news.scss

```diff
@@ -1026,6 +1025,22 @@
 				single-chain-children-stack: (
 					gap: var(--global-spacing-6),
 				),
+				tag-title: (
+					components: (
+						heading: (
+							font-family: var(--font-family-primary),
```

Comment:
Looking at all the changes that are being flagged with Chromatic - we should set the component `heading` to be using the `font-family: var(--font-family-primary)` - looks like a miss in previous work and that News Theme CSS is causing a little confusion for our storybook as the primary font is hard coded to be Arial for News Theme CSS.

Reply:
I saw you added this in your number list PR, so I made sure to remove that piece from the heading here so it wasn't duplicating styles.